modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/http/access/AccessLogger.java

```diff
@@ -211,6 +254,29 @@ protected synchronized void open() {
         }
     }
 
+    /*
```

Comment:
Incorrect java doc style, shall we correct?

Reply:
+1 filed